Okay, I did something wrong: 
```
$ ./model/bin/ww3_from_ftp.sh model 
[ERROR] input argument not recognized
        use -i for interactive mode
        use -k to keep tar files

```
If we have -k for keeping the tar files, do we still need to have the "y y"?  I thought that was the default behavior now.

--- a/model/bin/ww3_from_ftp.sh
+++ b/model/bin/ww3_from_ftp.sh
@@ -43,23 +43,23 @@ elif [ $# -gt 1 ] ; then
   exit 1
 fi
 
-model_dir=$(cd $(dirname $(dirname $0)) > /dev/null && pwd -P)
-echo $model_dir
-
+dir0=$(cd $(dirname $0) > /dev/null && pwd -P)
+ww3dir=$(dirname $(dirname $dir0))
+ 
 #Get top level directory of ww3 from user: 
 echo -e "\n\n This script will download data from the ftp for WAVEWATCH III "
 if [ "$interactive" = "n" ]
 then
-  echo $model_dir
+  echo $ww3dir
 else
-echo -e "Enter the absolute or relative path to the model directory, "
+echo -e "Enter the absolute or relative path to the main/top directory, "
 echo -e "this would be '../../' if in the model/bin directory "
 echo -e "or './' if already in the top/main directory:"
-  read model_dir 
+  read ww3dir 
 fi
 
 #Move to top level directory of ww3: 
-cd $model_dir 
+cd $ww3dir 
 
 #Download from ftp and uptar: 
 echo -e "Downloading and untaring file from ftp:" 
@@ -107,7 +107,7 @@ then
 else
   read keep
 fi
-if [ "${keep}" = "Y" ] || [ "${keep}" = "y" ]
+if [ "${keep}" = "N" ] || [ "${keep}" = "n" ]
 then
   echo -e '\n Deleting tar file ww3_from_ftp.${ww3ver}.tar.gz'
   rm ww3_from_ftp.${ww3ver}.tar.gz
@@ -123,7 +123,7 @@ then
 else
   read keep
 fi
-if [ "${keep}" = "Y" ] || [ "${keep}" = "y" ]
+if [ "${keep}" = "N" ] || [ "${keep}" = "n" ]
 then
   echo -e '\n Deleting the data_regtests folder'
   rm -rf data_regtests

--- a/model/bin/ww3_from_ftp.sh
+++ b/model/bin/ww3_from_ftp.sh
@@ -5,6 +5,18 @@
 #                                                    Created 0ct 10, 2017     #
 # --------------------------------------------------------------------------- #
 
+usage()
+{
+    echo ''
+    echo ' Usage : ./ww3_from_ftp.sh [options]'
+    echo ''
+    echo ' Options : '
+    echo '           -h : print usage'
+    echo '           -i : interactive mode'
+    echo '           -k : keep tar files'
+    echo ''
+}
+
 curr_dir=`pwd`
 
 # Set WW3 code version
@@ -13,20 +25,21 @@ ww3ver=v7.12.6
 interactive='n'
 keep='n'
 if [ $# -eq 1 ] ; then
-  if [ "$1" = "-i" ] ; then
+  if [ "$1" = "-h" ] ; then
+    usage
+    exit 0
+  elif [ "$1" = "-i" ] ; then
     interactive='y'
   elif [ "$1" = "-k" ] ; then
     keep='y'
   else
     echo '[ERROR] input argument not recognized'
-    echo '        use -i for interactive mode'
-    echo '        use -k to keep tar files'
+    usage
     exit 1
   fi
 elif [ $# -gt 1 ] ; then
   echo '[ERROR] only one input argument accepted'
-  echo '        use -i for interactive mode'
-  echo '        use -k to keep tar files'
+  usage
   exit 1
 fi
 

--- a/model/bin/ww3_from_ftp.sh
+++ b/model/bin/ww3_from_ftp.sh
@@ -35,13 +35,13 @@ echo $model_dir
 
 #Get top level directory of ww3 from user: 
 echo -e "\n\n This script will download data from the ftp for WAVEWATCH III "
-echo -e "Enter the relative path to the main/top level directory, this would "
-echo -e "be '../../' if in the model/bin directory or '.' if already in the "
-echo -e "top/main directory:"
 if [ "$interactive" = "n" ]
 then
   echo $model_dir
 else
+echo -e "Enter the absolute or relative path to the model directory, "
+echo -e "this would be '../../' if in the model/bin directory "
+echo -e "or './' if already in the top/main directory:"
   read model_dir 
 fi
 

